bigquery_etl/glean_usage/templates/cross_channel_events_unnested.view.sql

```diff
@@ -0,0 +1,16 @@
+-- Generated via ./bqetl glean_usage generate
+CREATE OR REPLACE VIEW
+    `{{ project_id }}.{{ target_view }}`
+AS
+{% for (dataset, channel) in datasets -%}
+{% if not loop.first -%}
+UNION ALL
+{% endif -%}
+{% if app_name == "fenix" -%}
+SELECT e.* EXCEPT (events) REPLACE(mozfun.norm.fenix_app_info("{{ dataset }}", client_info.app_build).channel AS normalized_channel), event.*
```

Comment:
As discussed in https://bugzilla.mozilla.org/show_bug.cgi?id=1710088#c2 we'd like these event fields to be prefixed with `event_`, so they'll have to be listed explicitly. We can probably factor that part out of the conditional so it doesn't have to be repeated.